src/main/java/baseball/Util.java

```diff
@@ -0,0 +1,59 @@
+package baseball;
+
+import camp.nextstep.edu.missionutils.Console;
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class Util {
+
+    public static final int BASE_COUNT = 3;
+
+    List<Integer> initComputer() {
+        List<Integer> computer = new ArrayList<>(BASE_COUNT);
+        while (computer.size() < BASE_COUNT) {
+            int randomNumber = Randoms.pickNumberInRange(1, 9);
+            if (!computer.contains(randomNumber)) {
+                computer.add(randomNumber);
+            }
+        }
+        return computer;
+    }
+
+    List<Integer> input() {
+        String input = Console.readLine();
+        validate(input);
+        return Arrays.stream(input.split("")).map(Integer::parseInt).collect(Collectors.toList());
```

Comment:
```suggestion
        return Arrays.stream(input.split(DELIMITER))
            .map(Integer::parseInt)
            .collect(toList());
```
스트림을 아주 잘 사용해 주셨어요!
일반적으로 스트림을 사용할때에는 위와같은 방식으로 가독성을 높혀줍니다. 
stream을 사용하는 이유중 중요한 요소는 `가독성이 좋다.` 도 있거든요! 이러한 이점을 살리기 위한 개행을 준수해 주는것도 좋아 보입니다!
또한 위 로직을 메서드로 분리할 수 있을 것 같아요!

- `""`의 경우로 구분자의 일종이니 상수로 만들어 보시는게 어떠신가요?
- Collectors.toList()와 같은 메서드을 그대로 사용하면 읽을 소스가 길어져서 가독성을 해칠 수 있습니다! static import에 대해서 알아보시면 좋을 것 같아요 :)

Reply:
감사합니다

또 꼼꼼히 체크하지 못하는 습관이 나온 것 같습니다.

개행준수와 함께 가독성을 향상한 코드를 짜기위해 노력하겠습니다!

""의 상수화와 static import, 개행 준수 적용시켜보겠습니다😀